linux/step05_centos7_nginx.sh

```diff
@@ -1,15 +1,19 @@
 #!/bin/bash
 
+#start-copy
+cp setup_omero_nginx.sh ~omero
+#end-copy
+
 #start-install
 yum -y --enablerepo=cr install nginx
```

Comment:
@jburel I am not sure if I follow but for C6 if this will install nginx  1.0.15. I am guessing is better to leave as it is (installing from Nginx repository)